packages/@aws-cdk/aws-events/test/test.rule.ts

```diff
@@ -49,6 +49,47 @@ export = {
     test.done();
   },
 
+  'get rate as token'(test: Test) {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, 'MyScheduledStack');
+    const lazyDuration = cdk.Duration.minutes(cdk.Lazy.number({ produce: () => 5 }));
+
+    new Rule(stack, 'MyScheduledRule', {
+      ruleName: 'rateInMinutes',
+      schedule: Schedule.rate(lazyDuration),
+    });
+
+    // THEN
+    expect(stack).toMatch({
+      'Resources': {
+        'MyScheduledRuleD15B5BA3': {
+          'Type': 'AWS::Events::Rule',
+          'Properties': {
+            'Name': 'rateInMinutes',
+            'ScheduleExpression': 'rate(5 minutes)',
+            'State': 'ENABLED',
+          },
+        },
+      },
+    });
```

Comment:
When asserting properties of single resources, we use -

```suggestion
    expect(stack).toHaveResourceLike('AWS::Events::Rule', {
      'Name': 'rateInMinutes',
      'ScheduleExpression': 'rate(5 minutes)',
    });
```